src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx

```diff
@@ -121,7 +111,10 @@ class IstioConfigDetailsPage extends React.Component<RouteComponentProps<IstioCo
 
   renderEditor = (istioObject: any) => {
     const yamlSource = yaml.safeDump(istioObject, safeDumpOptions);
-    const aceValidations = parseAceValidations(yamlSource, this.state.validations);
+    const aceValidations = this.state.istioObjectDetails
+      ? parseAceValidations(yamlSource, this.state.istioObjectDetails.validation)
+      : parseAceValidations(yamlSource, undefined);
```

Comment:
This looks odd, if not defined this.state.istioObjectDetails perhaps you can build an empty aceValidations object instead to invoke the parseAceValidations() with undefined.

Reply:
Well, the ``parseAceValidations()`` appeared to expect such behavior as it has ``validation?: ObjectValidation`` in the parameter list as well as ``if (!validation`` branch. So given that behavior I think the choice makes sense.